tg/decorators.py

```diff
@@ -329,7 +329,42 @@ class decode_params(object):
 
     By default the arguments are parsed in **JSON** format (which is
     currently the only supported format).
-
+    
+    This should be used like:
+        Image you are posting a payload of type 'application/json' like:
+        
+        ticket = {
+          ticketlist_id: 2,
+          typology: 'Xmas Discount',
+          quantity: 3,
+          price: 4.75
+        }
+        
+        Your controller's method should be something like
+        
+        .. code-block:: python
+        
+        @expose('json')
```

Comment:
Doesn't this need to be properly indented inside the `.. code-block::`?
See https://github.com/TurboGears/tg2/blob/eb81fd4bf5faa125edf0dfb6fd72407fead7efde/tg/support/registry.py#L77-L88 there the code is over indented from the `.. code-block:: python`